data/successor 1 introduction.txt

```diff
@@ -234,17 +239,17 @@ mission "Successors: First Contact 2"
 			choice
 				`	"I accept."`
 					goto accepted
-				`	"I do not accept."`
+				`	"I have to do whatever you want? I won't!"`
```

Comment:
I like giving the rejection option more emotion—I just think the phrasing of "whatever you want" doesn't feel quite right given that the scions have already established _precisely what_ they want.
Here's my idea for more specific phrasing, but if you have a different idea I'm happy to hear it.
```suggestion
				`	"You want me to do your errands? I won't!"`
```